src/components/CryptCard.tsx

```diff
@@ -0,0 +1,137 @@
+import {StyleSheet, Text, View, Image} from 'react-native';
+import React from 'react';
+import crypto from '../../metrics.json';
+import GainArrow from '../../assets/icons/north_east_black_24dp.svg';
+import LossArrow from '../../assets/icons/south_west_black_24dp.svg';
+
+type Props = {};
+
+const CryptCard = (Props: Props) => {
+  let percentageChange = (percentNumber: number) => {
+    let roundNumber = Math.round((percentNumber + Number.EPSILON) * 100) / 100;
+
+    if (roundNumber > 0) {
```

Comment:
redundant components:

```
const negative = roundNumber < 0

return (
<View style={styles.percentChangeText}>
          <GainArrow width={16} height={16} fill={negative ?  "red" : "green"} />
          <Text style={styles.textGain}>{(negative ? -roundNumber : roundNumber) + '%'}</Text>
        </View>
)
```

Reply:
also use styled components, a styled component can get props, so you can validate the color of the text inside the styles